embedded/bike.py

```diff
@@ -0,0 +1,120 @@
+import random
+from asyncio import sleep
+from datetime import timedelta
+
+from aiohttp.web_ws import WebSocketResponse
+from enum import Enum
+from marshmallow import ValidationError
+#from nacl.signing import SigningKey, SignedMessage
+import ed25519
+from embedded import logger
+from embedded.fona808 import GPSReading
+from .serializers import JsonRPCRequest, JsonRPCResponse
+from math import radians, cos, sin, asin, sqrt
+
+
+class Bike:
+    #In here we need whatever class is storing co-ordiantes
+    bid: int
+    seed: bytes
+    locked: bool
+    totalDistance: float
+    lockedLongitude: float
+    lockedLattitude: float
+    def __init__(self, bid, seed, locked=True):
+        self.bid = bid
+        self.seed = seed
+        self.signing_key = ed25519.SigningKey #Maybe?
```

Comment:
ed25519.SigningKey(seed)

you need to pass the seed in just like the previous impl. did